lib/index.js

```diff
@@ -40,6 +40,7 @@ VM.deps = {
  * @param {Boolean} opts.activatePrecompiles create entries in the state tree for the precompiled contracts
  * @param {Boolean} opts.allowUnlimitedContractSize allows unlimited contract sizes while debugging. By setting this to `true`, the check for contract size limit of 24KB (see [EIP-170](https://git.io/vxZkK)) is bypassed. (default: `false`; ONLY set to `true` during debugging)
  * @param {Boolean} opts.emitFreeLogs Changes the behavior of the LOG opcode, the gas cost of the opcode becomes zero and calling it using STATICCALL won't throw. (default: `false`; ONLY set to `true` during debugging)
+ * @param {Boolean} opts.enableEwasm enable EWASM precompiles. (default: `false`; Experimental feature)
```

Comment:
I'd name this `enableEwasmPrecompiles` or something similar, since it is not complete ewasm.